lib/builtin/primitive_ops.js

```diff
@@ -143,10 +183,26 @@ function likeTest(a, b) {
         a = a.normalize("NFD").replace(/[\u0300-\u036f]/g, "");
         b = b.toLowerCase();
         b = b.normalize("NFD").replace(/[\u0300-\u036f]/g, "");
+        a = a.replace(/[^\w\s]|_/g, ' ');
```

Comment:
This pattern will replace all non-ASCII characters, including CJK characters, Arabic characters, etc. We probably don't want that.
A better regex might be `/[\p{Mark}\p{Punctuation}\p{Separator}\p{Other}_]/u` (note the u at the end)